taccsite_cms/templatetags/tacc_uri_shortcuts.py

```diff
@@ -111,7 +111,8 @@ def get_menu_uri(menu_item):
             {{ child.get_menu_title }}</a>
             # <a href="https://otherwebsite.com/some-page">some page there</a>
     """
-    if hasattr(menu_item.attr, 'redirect_url'):
```

Comment:
This python was wrong. [I should use `in` operator to test for members of a dictionary.](https://stackoverflow.com/a/10724850/11817077) I do not know how this was tested successfully in https://github.com/TACC/Core-CMS/pull/453. I would bet I had an old working snippet solution active (and working) at the same time I was testing the new branch.

Reply:
This was deployed to BrainMap `pprd`, but overwritten by test deploy of [v3.8.1](https://github.com/TACC/Core-CMS/releases/tag/v3.8.1).